docs/machine-learning/tutorials/image-classification-api-transfer-learning.md

```diff
@@ -0,0 +1,399 @@
+---
+title: 'Tutorial: Automated visual inspection using transfer learning'
+description: This tutorial illustrates how to use transfer learning to train a TensorFlow deep learning model in ML.NET using the image detection API to classify images of concrete surfaces as cracked or not cracked.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 10/25/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to use ML.NET so that I can use transfer learning in an image classification scenario to classify images using a pretrained TensorFlow model and ML.NET's Image Classification API.
+---
+
+# Tutorial: Automated visual inspection using transfer learning with the ML.NET Image Classification API
+
+Learn how to train a custom deep learning model using transfer learning, a pretrained TensorFlow model and the ML.NET Image Classification API to classify images of concrete surfaces as cracked or uncracked.
```

Comment:
```suggestion
Learn how to train a custom deep learning model using transfer learning, a pretrained TensorFlow model and the [ML.NET Image Classification API](URL?) to classify images of concrete surfaces as cracked or uncracked.
```

Reply:
Provide a link for the Image Classification API? It sounds like an API separate from the ML.NET API, but I don't know where to find it.